docs/source/tutorials-and-examples/tutorial.rst

```diff
@@ -194,11 +194,11 @@ Now that you have your training code, you can package it so that other data scie
           channels:
             - defaults
           dependencies:
-            - numpy=1.14.3
-            - pandas=0.22.0
-            - scikit-learn=0.19.1
+            - python=3.6
+            - scikit-learn=0.23.2
+            - pip
             - pip:
-              - mlflow
+              - mlflow>=1.0
```

Comment:
Can we use sphinx's `literalinclude` directive here instead of hardcoding file content?

https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-literalinclude

Reply:
You can preview doc and makes sure it's rendered fine like this:

![Jan-15-2021 11-32-42](https://user-images.githubusercontent.com/17039389/104673633-78fa2d00-5725-11eb-880b-8957374c7271.gif)

Reply:
Haven't used this before. Giving it a shot. Will post an update.